lib/swagger-client.js

```diff
@@ -1460,22 +1460,8 @@ Operation.prototype.encodeQueryParam = function(arg) {
   return encodeURIComponent(arg);
 };
 
-/**
- * TODO revisit, might not want to leave '/'
- **/
 Operation.prototype.encodePathParam = function(pathParam) {
-  var encParts, part, parts, i, len;
```

Comment:
Replying [this](https://github.com/swagger-api/swagger-ui/pull/1015#discussion_r26013542). 

```js
return pathParam.toString().split('/').map(encodeURIComponent).join('/')
```

The above code does exactly the same thing

@fehguy should we merge this change?